internal/service/credentials/resource_credential_issuance_rule_test.go

```diff
@@ -161,68 +210,6 @@ func TestAccCredentialIssuanceRule_Full(t *testing.T) {
 	})
 }
 
-func TestAccCredentialIssuanceRule_InvalidConfigs(t *testing.T) {
```

Comment:
Is there a reason why these tests have been removed?

Reply:
I felt like they were outliers when compared to the rest of the provider, and were maybe overkill and adding unnecessary checks.  I'm 100% fine with adding them back.